DXRModules/DeusEx/Classes/DXRStartMap.uc

```diff
@@ -523,14 +524,20 @@ static function bool BingoGoalImpossible(string bingo_event, int start_map, int
     case 2: // NSF Generator
         switch(bingo_event)
         {
+        case "LeoToTheBar":
```

Comment:
LeoToTheBar and PresentForManderley are both actually covered below in the more general switch statement